### PR TITLE
trying things

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: lib/src/vector_math/plane.dart [hunk 1]
```diff
@@ -6,7 +6,7 @@ part of vector_math;
 
 class Plane {
   final Vector3 _normal;
-  double _constant;
+  double constant;
```
Comment (kevmoo) [inline lib/src/vector_math/plane.dart]: @rakudrama – FYI – I turned a few things that were properties into fields – assuming that's okay

Comment (rakudrama) [inline lib/src/vector_math/plane.dart]: Why this change?
It is a public field - visible outside the package, which will hurt dart2js type inference around dynamic call sites.

Comment (kevmoo) [reply lib/src/vector_math/plane.dart]: it was a property (see below) – nixing the pointless property is goodness, right?

File: lib/src/vector_math/plane.dart [hunk 2]
```diff
@@ -34,40 +34,36 @@ class Plane {
   }
 
   Vector3 get normal => _normal;
-  double get constant => _constant;
```
Comment (kevmoo) [inline lib/src/vector_math/plane.dart]: @rakudrama !
